pkgs/development/python-modules/tensorflow/bin.nix

```diff
@@ -61,14 +61,21 @@ in buildPythonPackage rec {
   # bleach) Hence we disable dependency checking for now.
   installFlags = lib.optional isPy36 "--no-dependencies";
 
+
+  # Upstream has a pip hack that results in bin/tensorboard being in both tensorflow
+  # and the propageted input tensorflow-tensorboard which causes environment collisions.
+  #
+  # https://github.com/tensorflow/tensorflow/blob/v1.7.1/tensorflow/tools/pip_package/setup.py#L79
+
   # Note that we need to run *after* the fixup phase because the
   # libraries are loaded at runtime. If we run in preFixup then
   # patchelf --shrink-rpath will remove the cuda libraries.
   postFixup = let
     rpath = stdenv.lib.makeLibraryPath
       ([ stdenv.cc.cc.lib zlib ] ++ lib.optionals cudaSupport [ cudatoolkit_joined cudnn nvidia_x11 ]);
-  in
-  lib.optionalString (stdenv.isLinux) ''
+  in ''
+    rm $out/bin/tensorboard $out/bin/.tensorboard-wrapped
```

Comment:
Can we also just do `rm $out/bin/tensorboard` in `postInstall`?

Reply:
That's cleaner all right.  Thanks!  I've pushed a new version.